software/controller/controller.sh

```diff
@@ -115,7 +115,7 @@ install_linux() {
                clang-tidy
   pip3 install -U pip
   pip3 install codecov pyserial matplotlib pandas gitpython
-  pip3 install platformio==5.1.1
```

Comment:
does this need to be updated in some readmes as well?

Reply:
It is already up to date in `software\controller\README.md`. I don't see another place where we reference the `platformio` version.